healthcare/dicom_test.go

```diff
@@ -86,14 +99,92 @@ func TestDICOMStore(t *testing.T) {
 	// })
 
 	testutil.Retry(t, 10, 2*time.Second, func(r *testutil.R) {
-		if err := dicomWebRetrieveStudy(ioutil.Discard, tc.ProjectID, location, datasetID, dicomStoreID, "studies/1.3.6.1.4.1.11129.5.5.111396399361969898205364400549799252857604"); err != nil {
-			r.Errorf("dicomWebSearchInstances got err: %v", err)
+		// Remove the output file if it already exists.
+		os.Remove(studyOutputFile)
+		buf.Reset()
+
+		err := dicomWebRetrieveStudy(buf, tc.ProjectID, location, datasetID, dicomStoreID, studyUID, studyOutputFile)
+		if err != nil {
+			t.Errorf("Testing dicomWebRetrieveStudy: %v", err)
```

Comment:
No need for "Testing".

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures (OK to not print args here since it's not a table test).

Reply:
Done. You still need to print err, correct?